test/gen-template/matter/external-addon-helper.js

```diff
@@ -0,0 +1,31 @@
+/**
+ *
+ *    Copyright (c) 2021 Silicon Labs
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+// This is an example of an external addon helper for templates.
+
+async function test_external_addon_helper(api, context) {
+  //add queries here
+  return 'This is example of test external addon helper.'
```

Comment:
You have a very simplistic unit test, simply returning a constant string. You also bind function, so I'm very lost as to what is the actual API and context that is passed to this function.

Can you create a unit test that uses some queries from the `api` and gets some data from the context?

Also note, that the helpers have their own "hash" properties passed. It would be nice to see if that mechanism works fully.

Suggestion:
   create a helper that actually does something real-like, like retrieve a "longest cluster name" or something similar, and then test it with that data. Then also make it add a `key="value"` type of argument, that will modify its behavior.

Reply:
Okay will do

Reply:
done